lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -69,15 +70,21 @@ export default function StudentSelector({
     );
 
     return (
-      <span className="StudentsSelector__students">
+      <span className="relative">
         {/*
         This lint issue may have arisen from browser inconsistency issues with
         `onChange` which have since been fixed. See browser compatibility note here:
         https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event#annotations:xeC6ClQsEequhUdih2lXzw
         */}
         {/* eslint-disable-next-line jsx-a11y/no-onchange*/}
         <select
-          className="StudentsSelector__students-select"
+          className={classNames(
+            'appearance-none w-full h-touch-minimum',
+            'pl-4 pr-8', // Make room on right for custom down-caret Icon
+            'xl:w-80', // Fix the width at wider viewports
```

Comment:
I found a pity that the width of the `select` is wider for large viewports than extra-large viewports.

Reply:
Good feedback! Might take another look at this, or tune it in a follow-up.